### PR TITLE
Introducing Az.Security SecurityTopology cmdlets

### PR DESCRIPTION
**Pull request by ariklin**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

Design review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/613

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Security/Security/Cmdlets/AllowedConnections/GetAllowedConnections.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------
+using System.Linq;
+using System.Management.Automation;
+using Commands.Security;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.Security.Models.AllowedConnections;
```
Comment (VeryEarly) [inline src/Security/Security/Cmdlets/AllowedConnections/GetAllowedConnections.cs:19]: looks like this allowedconnections was not defined yet

Comment (ariklin) [reply src/Security/Security/Cmdlets/AllowedConnections/GetAllowedConnections.cs:19]: Sorry about that. I will remove it.

Comment (ariklin) [reply src/Security/Security/Cmdlets/AllowedConnections/GetAllowedConnections.cs]: Removed.

**Pull request merged**